test/Microsoft.TestPlatform.Common.UnitTests/Filtering/FastFilterTests.cs

```diff
@@ -94,14 +94,19 @@ public void FastFilterWithSingleEqualsClause()
     [TestMethod]
     public void ValidForPropertiesHandlesBigFilteringExpressions()
     {
-        StringBuilder testCaseFilter = new("Test1");
+        StringBuilder testCaseFilter = new("Category=Test1");
+
         for (int i = 0; i < 1e5; i++)  // creating a 100k filter cases string
         {
             testCaseFilter.Append("|Test2");
         }
 
         var filterExpressionWrapper = new FilterExpressionWrapper(testCaseFilter.ToString());
-        Assert.IsNull(filterExpressionWrapper.ValidForProperties(new List<string>() { "FullyQualifiedName" }, null));
+        string[]? invalidProperties = filterExpressionWrapper.ValidForProperties(new List<string>() { "FullyQualifiedName" }, null);
+
+        Assert.IsNotNull(invalidProperties);
+        Assert.AreEqual(invalidProperties?.Count(), 1);
+        Assert.AreEqual(invalidProperties![0], "Category");
```

Comment:
As said on previous comment, I don't think you need null validation here.

```suggestion
        Assert.AreEqual(invalidProperties[0], "Category");
```